doc/user-guide-v1.adoc

```diff
@@ -4,6 +4,65 @@ The Open Liberty Operator can be used to deploy and manage applications running
 
 **Important**: This user guide only applies to Operator versions **1.2.0 and above**.
 
+== System requirements
+
+Your environment must meet cluster, sizing, persistent storage, and network requirements for Open Liberty operator.
+
+=== OpenShift Container Platform Requirements
+If you are installing a Open Liberty operator on an Red Hat OpenShift cluster, your environment must meet Red Hat OpenShift Container Platform (OCP) cluster requirements.
+
+==== OCP requirements
+
+Open Liberty operator requires an link:++https://docs.openshift.com/container-platform/4.15/welcome/index.html++[OCP version 4.15], link:++https://docs.openshift.com/container-platform/4.14/welcome/index.html++[OCP version 4.14], or link:++https://docs.openshift.com/container-platform/4.12/welcome/index.html++[OCP version 4.12] cluster on Linux x86_64 (amd64), Linux on Power (ppc64le), or Linux on IBM Z (s390x) platform, with cluster-admin permissions. To manage OCP projects with OCP CLI (oc) commands, the installation also requires the link:++https://docs.openshift.com/container-platform/4.9/cli_reference/openshift_cli/getting-started-cli.html++[OCP CLI].
```

Comment:
Add OCP version 4.16 (https://docs.openshift.com/container-platform/4.16/welcome/index.html) to the list